ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -100,8 +106,26 @@ private[sandbox] object SqlLedger {
         _ <- Resource.fromFuture(
           new FlywayMigrations(jdbcUrl).migrate()
         )
+        dbType = DbType.jdbcType(jdbcUrl)
+        storageBackendFactory = StorageBackendFactory.of(dbType)
         ledgerEndCache = MutableLedgerEndCache()
-        dao <- ledgerDaoOwner(ledgerEndCache, servicesExecutionContext, errorFactories).acquire()
+        dbDispatcher <- DbDispatcher
+          .owner(
+            dataSource =
+              storageBackendFactory.createDataSourceStorageBackend.createDataSource(jdbcUrl),
+            serverRole = serverRole,
+            connectionPoolSize = dbType.maxSupportedWriteConnections(databaseConnectionPoolSize),
+            connectionTimeout = databaseConnectionTimeout,
+            metrics = metrics,
+          )
+          .acquire()
+        dao = ledgerDaoOwner(
```

Comment:
The same as above - rename to `ledgerDao()`

Reply:
good point, will do in upcoming commits (to gain some speed now)